proto/zitadel/action/v2beta/action_service.proto

```diff
@@ -394,51 +471,123 @@ service ZITADELActions {
 }
 
 message CreateTargetRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
+  string name = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 1000},
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
+      example: "\"ip_allow_list\"";
+      min_length: 1
+      max_length: 1000
     }
   ];
-  Target target = 2 [
-    (validate.rules).message = {
-      required: true
+  // Defines the target type and how the response of the target is treated.
+  oneof target_type {
+    option (validate.required) = true;
+    RESTWebhook rest_webhook = 2;
+    RESTCall rest_call = 3;
+    RESTAsync rest_async = 4;
+  }
+  // Timeout defines the duration until ZITADEL cancels the execution.
+  // If the target doesn't respond before this timeout expires, then the connection is closed and the action fails.
```

Comment:
```suggestion
  // If the target doesn't respond before this timeout expires, then the connection is closed and the action fails. Depending on the target type and possible setting on `interrupt_on_error` following targets will not be called. In case of a `rest_async` target only this specific target will fail, without any influence on other targets of the same execution.
```
am i correct on this?

Reply:
Added